tests/scripts/task_rust.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+export LD_LIBRARY_PATH=lib:${LD_LIBRARY_PATH}
+export PYTHONPATH=python:nnvm/python:topi/python
```

Comment:
The test was not successful

NOTE you are using a relative path here. after you cd into rust, the PYTHONPATH is no longer correctly, try to run this script inside the docker

Reply:
you need to chmod +x tests/scripts/task_rust.sh